### DISCUSSION TITLE
Simple way to issue a command to restart the VPN.

### DISCUSSION THREAD

**Discussion by sboger** [qdm12/gluetun#1747] [Q&A] [answered]
I am looking for a simple way to trigger a vpn disconnect and reconnect. Not restart the container, just a vpn restart like when the healthcheck fails. I want to issue this a few times a week to randomly change the VPN endpoint.

For instance, I use this snippet to check the current location of the VPN:

`docker exec -ti [container id] 'wget' '-qO-' 'https://ipinfo.io' | jq -r '.city + ", " + .country'
`

Is there a similar command to issue inside of the container to perform a restart? Or maybe a curl request to the HTTP control server?

**Comment by sboger** [comment, marked as answer]
Answering my own question. One can utilize a "bug" (per qdm12) that triggers auto-healing to rotate to a new vpn endpoint by issuing this to the control server:

docker exec -ti [gluetun container id] 'wget' '-qO-' '--method=PUT' '--body-data={"status":"stopped"}' 'http://127.0.0.1:8000/v1/openvpn/status'

**Reply by awexandrr** [reply]
for wireguard, use `/v1/vpn/status` endpoint. https://github.com/qdm12/gluetun/issues/2277#issuecomment-2115964521

**Comment by ezekieldas** [comment]
Hey @sboger --A few questions, if you a have a moment I'd really appreciate your input.

- Are you using a wireguard or openvpn config?
- What gluetun version are you using? gluetun:v3 or earlier?
- You mentioned a 'bug (per qdm12)' --would it be either of these:
    - https://github.com/qdm12/gluetun/issues/2217
    - https://github.com/qdm12/gluetun/issues/2277

I'm wanting to collect some more detail on this and other updater issues.

Also, as noted in the issues I've mentioned, there is a deadly combination of using this to reissue an ipaddr / gateway if your cocktail involves:  > gluetun:v3 + wireguard + `/v1/openvpn/status`

I just summarized my previous use, etc. in this recent comment: 

https://github.com/qdm12/gluetun/issues/2277#issuecomment-2110797537

**Comment by yuoter** [comment]
@sboger I also found such option, it works a bit faster:
GET request to URI /openvpn/actions/restart
